assets/queries/terraform/aws/s3_bucket_with_all_permissions/query.rego

```diff
@@ -45,4 +45,5 @@ all_permissions(policyValue) {
 
 	common_lib.is_allow_effect(statement)
 	common_lib.containsOrInArrayContains(statement.Action, "*")
+	common_lib.containsOrInArrayContains(statement.Principal, "*")
```

Comment:
I think we should also look to `aws_s3_bucket_policy` as `assets\queries\terraform\aws\s3_bucket_allows_all_actions_from_all_principals` was doing